templates/angular/igx-ts/custom-templates/tree-grid-finjs/files/src/app/__path__/__name__.component.scss

```diff
@@ -0,0 +1,234 @@
+        }
+    }
+    .strongPositive1 {
+        background: #459a55;
+        color: #fff;
+    }
+    .strongPositive2 {
+        .igx-grid__td-text {
+            border-right: 4px solid #459a55;
+            padding-right: 4px;
+        }
+    }
+    // negative
+    .strongNegative {
+        .igx-grid__td-text {
+            background: #d31642;
+            color: #fff;
+        }
+    }
+    .strongNegative1 {
+        background: #d31642;
+        color: #fff;
+    }
+    .strongNegative2 {
+        .igx-grid__td-text {
+            border-right: 4px solid #d31642;
+            padding-right: 4px;
+        }
+    }
+    // selected
+    .igx-grid__td--selected.strongPositive1,
+    .igx-grid__td--selected.strongPositive2,
+    .igx-grid__td--selected.strongPositive {
+        background-color: #459a55 !important;
+        .fintech-icons,
+        .igx-grid__td-text {
+            color: #fff;
+        }
+    }
+    .igx-grid__td--selected.strongNegative1,
+    .igx-grid__td--selected.strongNegative2,
+    .igx-grid__td--selected.strongNegative {
+        background-color: #d31642 !important;
+        .fintech-icons,
+        .igx-grid__td-text {
+            color: #fff;
+        }
+    }
+    .controls-holder {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        flex-wrap: wrap;
+    }
+    .switches {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        flex: 1 0 0%;
+        min-width: 500px;
+        padding-right: 20px;
+        font-size: 0.9rem;
+        margin-bottom: -20px;
+    }
+    .control-item {
+        padding-right: 20px;
+    }
+    .igx-slider,
+    .igx-slider--disabled {
+        height: 24px;
+    }
+    .fintech-slider {
+        width: 40%;
+        min-width: 145px;
+    }
+    .fintech-play-controls {
+        width: 45%;
+        min-width: 500px;
+        margin-top: 20px;
+    }
+    .sample-toolbar {
+        height: 20px;
+        font-size: 0.8rem;
+        line-height: 20px;
+    }
+    .fin-dark-theme {
+        .fintech-slider,
+        .sample-toolbar {
+            color: rgba(255, 255, 255, 0.87);
+        }
+    }
+    .igx-grid__outlet span,
+    .igx-excel-filter span,
+    .igx-excel-filter header,
+    .igx-excel-filter input {
+        font-size: 0.8125rem;
+    }
+    // .igx-button-group > .igx-button--compact {
+    //     height: 1rem;
+    //     min-height: 1.5rem;
+    // }
+    // .igx-button-group > .igx-button--cosy {
+    //     height: 1rem;
+    //     min-height: 1.875rem;
+    // }
+}
+
+.grid__wrapper {
+    padding: 5px 15px;
+    height: 100%;
+}
+
+ // Custom Dark Theme
+
+ // Import the theme utilities first
+@import '~igniteui-angular/lib/core/styles/themes/utilities';
+@import '~igniteui-angular/lib/core/styles/themes/index';
+@include igx-core();
+@include igx-theme($default-palette);
+
+ @import '../themes/mixins';
+
+ @include scrollbar-love();
```

Comment:
move this include inside of the below `::ng-deep` selector, so that the scrollbars are properly affected